include/openssl/core_dispatch.h

```diff
@@ -176,6 +176,10 @@ OSSL_CORE_MAKE_FUNC(int, BIO_vsnprintf,
 OSSL_CORE_MAKE_FUNC(int, BIO_ctrl, (OSSL_CORE_BIO *bio,
                                     int cmd, long num, void *ptr))
 
+/* New seeding functions prototypes with the 101-104 series */
+#define OSSL_FUNC_GET_USER_ENTROPY            98
```

Comment:
Add to doc/man7/provider-base.pod

Maybe use 100+ for these and shift them down after the others..

Reply:
We cannot change the others, they are part of the public API.  I could grab a group after them, but that seems like a worse solution.

Reply:
Added but not renumbered.

Reply:
Added to the doc file but not renumbered.